model/simulationtests/autosize_hvac.rb

```diff
@@ -1047,6 +1064,16 @@ def curve_quadratic(model, c_1constant, c_2x, c_3xPOW2, minx, maxx, miny, maxy)
     chw_loop.addDemandBranchForComponent(panel_coil)
     zoneHVACCoolingPanelRadiantConvectiveWater.addToThermalZone(zn)
 
+  when 41
+    plhp_airsource_htg = OpenStudio::Model::HeatPumpAirToWaterFuelFiredHeating.new(model)
+    plhp_airsource_htg.autosizeNominalHeatingCapacity
+    plhp_airsource_htg.autosizeDesignFlowRate
+    plhp_airsource_htg.autosizeDesignTemperatureLift
+
+    plhp_airsource_clg = OpenStudio::Model::HeatPumpAirToWaterFuelFiredCooling.new(model)
+    plhp_airsource_clg.autosizeNominalCoolingCapacity
+    plhp_airsource_clg.autosizeDesignFlowRate
+    plhp_airsource_clg.autosizeDesignTemperatureLift
```

Comment:
Why is this here?

Reply:
That wasn't the reason I deleted this block. Above you have almost the same, except you then add them to a PlantLoop (as they should).

This block here achieves nothing but to create two extras orphaned objects. This is meant to add a ZoneHVAC component on a zone (the when 41 means the 41st zone)

Reply:
Ah. I must have stubbed out (and didn't finish) adding a ZoneHVAC component (like in heatpump_airtowater_fuelfired.rb).